src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -122,97 +123,119 @@ const WorkflowDashboard = _.flow(
 
   return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
     workflowDetailsBreadcrumbSubtitle(namespace, name, submissionId, workflowId),
-    workflow === undefined ?
-      h(Fragment, [
+    Utils.cond([
+      workflow === undefined,
+      () => h(Fragment, [
         div({ style: { fontStyle: 'italic', marginBottom: '1rem' } }, [`Fetching workflow metadata...`]),
         centeredSpinner()
-      ]) :
-      h(Fragment, [
+      ])
+    ], [
+      metadataArchiveStatus === 'ArchivedAndDeleted',
+      () => h(Fragment, [
         div({ style: { fontStyle: 'italic', marginBottom: '1rem' } }, [`Workflow metadata fetched in ${fetchTime}ms`]),
-        div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
-          makeSection('Workflow Status', [
-            div({ style: { lineHeight: '24px', marginTop: '0.5rem' } }, [makeStatusLine(style => statusIcon(status, style), status)])
-          ]),
-          makeSection('Workflow Timing', [
-            div({ style: { marginTop: '0.5rem', display: 'grid', gridTemplateColumns: 'auto 1fr', gap: '0.5rem' } }, [
-              div({ style: styles.sectionTableLabel }, ['Start:']), div([start ? Utils.makeCompleteDate(start) : 'N/A']),
-              div({ style: styles.sectionTableLabel }, ['End:']), div([end ? Utils.makeCompleteDate(end) : 'N/A'])
-            ])
+        div({ style: { display: 'flex', flexDirection: 'column' } }, [
+          div({ style: { lineHeight: '24px', marginTop: '0.5rem', display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+            icon('info-circle', { size: 18, style: { color: colors.accent(), marginRight: '0.5rem' } }),
+            div({ style: Style.elements.sectionHeader, paddingRight: '0.5rem' }, ' Workflow Details Archived')
```

Comment:
This `paddingRight` needs to be passed in the `style` object to take effect (but we should probably just remove it).